lib/tasks/ee.rake

```diff
@@ -0,0 +1,263 @@
+require 'yaml'
+require 'erb'
+
+class Database
+  RECOGNIZED_OPTIONS = %w(user host port socket)
+
+  def initialize(options = {})
+    @options = options
+  end
+
+  def export_to(filename)
+    run "#{inline_password}#{find_executable('mysqldump')} #{command_flags} #{database} > #{filename}"
+  end
+
+  def import_from(filename)
+    run "#{inline_password}#{find_executable('mysql')} #{command_flags} #{database} < #{filename}"
+  end
+
+  def drop
+    run "#{inline_password}#{find_executable('mysqladmin')} #{command_flags} -f drop #{database}"
+  end
+
+  def create
+    execute "CREATE DATABASE IF NOT EXISTS #{database}"
+  end
+
+  def execute(command, database = nil)
+    full_command = (database.nil?) ? command : "USE `#{database}`; #{command}"
+
+    run "#{inline_password}#{find_executable('mysql')} #{command_flags} -e '#{full_command}'"
+  end
+
+  def db_execute(command)
+    execute command, database
+  end
+
+  private
+
+  def run(command)
+    puts command
+    `#{command}`
+  end
+
+  def database
+    @options['database']
+  end
+
+  def inline_password
+    @options['password'] ? "MYSQL_PWD=#{@options['password']} " : ''
+  end
+
+  def command
+    "#{executable} #{flags.join(' ')}"
+  end
```

Comment:
This method is never run (as evidenced by there being no `#executable` method).

In general, this file is messy and could be cleaned up. There's a lot of duplication in building the command (inlining password, finding executable, including flags, etc). I know you're just porting this from another thing, but worth going through this to clean up the code?